pipeline/vax/rpt/main.py

```diff
@@ -0,0 +1,44 @@
+import pandas as pd
+from google.cloud import storage
+import matplotlib.pyplot as plt
+
+def generate_vax_report(_):
+
+    # URL of the csv used directly
+
+    df = pd.read_csv("https://www.dropbox.com/sh/y949ncp39towulf/AACd3YxzfB0fHkjQ1YJG-W2ba/covid/csv/covid_vaccination.csv?dl=1")
+
+    # convert date to proper format
+    df["date"] = pd.to_datetime(df["date"])
+
+    statesList = df["state"].unique()
+    for state in statesList:
+        stateDF = df.loc[df["state"] == state]
+
+        # state level total aggregates
+        totalStateDailyAgg = stateDF.groupby("date").sum()
+
+        plt.subplots(1,1)
+        totalStateDailyAgg["first_dose_admin"].plot()
+        plt.savefig("/tmp/first_dose_admin_{}.png".format(state))
```

Comment:
can we add logic to check the filesize? since the renderer runs in its own thread and delegates to disk I/O we're not always sure the image has finished saving - check the other twitter image generation script for examples

Reply:
I just added these plots as placeholders for the time being. I'll change them to the convention used